frontend/src/app/problems/add/execution-constraints.tsx

```
@@ -0,0 +1,67 @@
+                            <TableCell>Time Limit</TableCell>
+                            <TableCell>Memory Limit</TableCell>
+                        </TableRow>
+                    </TableHead>
+                    <TableBody>
+                        {selectedLanguages.map((language) => (
+                            <TableRow key={language}>
+                                <TableCell>{language}</TableCell>
+                                <TableCell>
+                                    <TextField
+                                        value={timeLimit}
+                                        onChange={(e) => setTimeLimit(parseInt(e.target.value))}
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)